indico/modules/logs/__init__.py

```diff
@@ -26,6 +26,13 @@ def _extend_event_management_menu(sender, event, **kwargs):
     return SideMenuItem('logs', _('Logs'), url_for('logs.event', event), section='reports')
 
 
+@signals.menu.items.connect_via('user-profile-sidemenu')
+def _extend_profile_sidemenu(sender, user, **kwargs):
+    if not session.user.is_admin:
+        return
+    return SideMenuItem('logs', _('Logs'), url_for('logs.user'))
```

Comment:
Set the weight to something like `-100` so the entry shows up last. (Data export is weight 0, everything else is positive)